### DISCUSSION TITLE
Is my website lost if my device is destroyed.

### DISCUSSION THREAD

**Discussion by ashlamp08** [beakerbrowser/beaker#1760] [General]
Can i recover ownership to my website if my device is destroyed? I assume these sites are signed by some key to make it modifiable(modify and attribute to the same old address) only by me and forked sites will have different address.

In the case of device destruction, can i login to different device and still work on my websites?
Also how do i recover any other private data stored?

**I am quite new to P2P in general.**

**Comment by pfrazee** [comment]
@ashlamp08 You understand things pretty well. We don't yet have a backup solution, and manual backup is somewhat challenging due to some properties of the network which could lead to data-corruption if done incorrectly.

We'll keep everyone updated on when we solve this.
